int64 level
int64 searchPos
---
bool success
